kobo/apps/hook/views/hook_log.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import json
+
+from django.shortcuts import get_object_or_404
+from django.utils.translation import ugettext as _
+from rest_framework import viewsets, mixins, status
+from rest_framework.decorators import detail_route
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework_extensions.mixins import NestedViewSetMixin
+
+from ..models.hook_log import HookLog
+from ..serializers.hook_log import HookLogSerializer
+from kpi.models import Asset
+from kpi.serializers import TinyPaginated
+from kpi.views import AssetOwnerFilterBackend, SubmissionViewSet
+
+
+class HookLogViewSet(NestedViewSetMixin,
+                     mixins.RetrieveModelMixin,
+                     mixins.ListModelMixin,
+                     viewsets.GenericViewSet):
+    """
+    ## Logs of an external service
+
+    ** User can't add, update or delete logs with the API. He can only retry failed attemps (see below)**
```

Comment:
Please replace with `Users can't.... They can only...`. It's worthwhile to have gender-neutral language whenever possible. (I was going to say: maybe this isn't a concern in French, but [clearly I would've been mistaken](https://www.theatlantic.com/international/archive/2017/11/inclusive-writing-france-feminism/545048/)!)

Oh, and `attempts` is missing a `t` :wink:

Reply:
Gotcha!